File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
       int n;
   do
   {
        n = get_int("Height:");
   }
   while(n>8 || n<1);

    for(int i = 0 ; i < n ; i++)
    {
        for (int k =1 ; k <n-i; k++)
        {
            printf(" ");
        }
        for (int j = 0 ; j < i +1 ; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int l = 0 ; l < i +1 ; l++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: sentimental-mario-more/mario.py
from cs50 import get_int

h = 0

while h < 1 or h >8 :
    h = get_int("height : ")

for i in range(1,h+1):
    f = h - i
    print(" "*f, end ="")
    print("#"*i, end ="")
    print(" "*2, end ="")
    print("#"*i)
